components/post_view/commented_on/commented_on.jsx

```diff
@@ -32,24 +32,33 @@ export default class CommentedOn extends PureComponent {
     makeUsername = () => {
         const postProps = this.props.post.props;
         let username = this.props.displayName;
-        if (
-            this.props.enablePostUsernameOverride &&
-            postProps &&
-            postProps.from_webhook === 'true' &&
-            postProps.override_username
-        ) {
+        if (this.props.enablePostUsernameOverride && postProps && postProps.from_webhook === 'true' && postProps.override_username) {
             username = postProps.override_username;
         }
         return username;
     }
 
-    render() {
-        const {
-            post,
-        } = this.props;
+    makeCommentedOnMessage = () => {
+        const {post} = this.props;
+        let message = '';
+        if (post.message) {
+            message = Utils.replaceHtmlEntities(post.message);
+        } else if (post.file_ids && post.file_ids.length > 0) {
+            message = (
+                <CommentedOnFilesMessage parentPostId={post.id}/>
+            );
+        } else {
```

Comment:
Just to be safe, maybe have this be `} else if (post.props && post.props.attachments && post.props.attachments.length > 0) {`? While we require a message or files on a post in the UI, I wouldn't be surprised if there's a way to create a post that doesn't have a message or files or attachments